docs/v0.5/getting-started/gke.md

```diff
@@ -70,7 +70,8 @@ gcloud services enable \
 
 ## Create a GKE cluster
 
-Choose a new unique lowercase cluster name and create the cluster. For this demo, three nodes should be sufficient.
+Choose a new unique lowercase cluster name and create the cluster. For this demo, three nodes should be sufficient. v0.5 of riff requires Kubernetes v1.15 or later.
```

Comment:
the gcloud command should be updated to create a compatible cluster. `--cluster-version=1.15` (`--cluster-version=latest` is still 1.13)

Reply:
I just tried it and i got v1.15.8
Is is possible that it's not consistent across regions?

Reply:
according to https://cloud.google.com/kubernetes-engine/versioning-and-upgrades

> `latest`: Specifies the highest supported Kubernetes version currently available on GKE in the cluster's zone or region.

`$ gcloud container get-server-config --zone $GCP_ZONE` will list the available versions for that zone.

Reply:
yep, looks like `latest` is 1.15 now. I'm not quite sure what latest means as 1.13 is still the default, and even while 1.15 was available, latest would give you an older cluster.

Reply:
I always thought that `latest` worked well, gave me the latest listed in the UI which was always more recent than the default version